source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
+
+Some of our best friends are white men, and there is nothing wrong with
+us^wthem, but all the same some of best friends are not, and for some reason the
+ones who aren't didn't come forward to speak at PyCon.
```

Comment:
This is a real bike shed, but: do we only want under-represented groups because they're our best friends? I feel like that's limiting: maybe consider changing "our best friends" to "our community members" or something similar?